test-harness/src/test/java/com/netflix/conductor/test/utils/MockExternalPayloadStorage.java

```diff
@@ -168,4 +168,19 @@ public Map<String, Object> downloadPayload(String path) {
         }
         return new HashMap<>();
     }
+
+    public Map<String, Object> createLargePayload(int repeat) {
```

Comment:
Does this need to be a parameterized method? Why not use a static large payload?

Reply:
yes could do that as well, but I find this way is more flexible to get the payload size I want for different tests. Like in this example, the fork join task only spawn 2 tasks, so I can play around with the parameter there to make sure the join output exceed the size limit. If for other tests, there are more tasks spawn from fork join, I could always use different parameter to achieve the purpose instead of having to create precise static large payload for different workflows